docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md

```diff
@@ -0,0 +1,60 @@
+---
+description: "Deploy HashiCorp Nomad Clients Cluster, a flexible scheduling and orchestration for diverse workloads, on Linode Compute Instances.'"
+keywords: ['HashiCorp','Nomad','Scheduling', 'orchestration']
+tags: ["marketplace", "linode platform", "cloud manager"]
+published: 2023-09-11
+modified_by:
+  name: Linode
+title: "Deploy HashiCorp Nomad Clients Cluster through the Linode Marketplace"
+authors: ["Linode"]
+---
+
+[HashiCorp Nomad](https://www.nomadproject.io/) is a workload scheduler and orchestrator that allows you to deploy and manage containerized or non-containerized, legacy applications using a single, unified workflow. Nomad uses bin packaging for efficient job scheduling and resource optimization, as well as enables developers to use declarative infrastructure-as-code for their deployments.
+
+{{< note type="warning" title="Marketplace App Cluster Notice" >}}
+This Marketplace App deploys 3, 5 or 7 Compute Instances as clients to horizontally scale an *existing HashiCorp Nomad Cluster*. The plan type and size you select is applied to each individual instance.
+
+Please be aware that each Compute Instance will appear on your invoice as a separate item. If you would rather deploy HashiCorp Nomad on a single Compute Instance, see [Deploy Hashicorp Nomad through the Linode Marketplace](/docs/products/tools/marketplace/guides/hashicorp-nomad/).
+{{< /note >}}
+
+## Deploying a Marketplace App
+
+{{< content "deploy-marketplace-apps-shortguide">}}
+
+{{< content "marketplace-verify-standard-shortguide">}}
+
+{{< note >}}
+**Estimated deployment time:** Nomad Clients Cluster should be fully installed within 15-20 minutes after the Compute Instances have finished provisioning.
+{{< /note >}}
+
+## Configuration Options
+
+- **Supported distributions**: Ubuntu 22.04 LTS
+- **Recommended plan**: Nomad clients can be of any size.
+- **Region**: The region you select should be the same as your existing Nomad Cluster.
+
+{{< content "marketplace-special-character-limitations-shortguide">}}
+
+### Nomad Client Options
+
+- **Linode API Token** *(required)*: Your API token is used to deploy additional Compute Instances as part of this cluster. At a minimum, this token must have Read/Write access to *Linodes*. If you do not yet have an API token, see [Get an API Access Token](/docs/products/tools/api/guides/manage-api-tokens/) to create one.
+
+- **Limited sudo user** *(required)*: A limited user account with sudo access is created as part of this cluster deployment. Enter your preferred username for this limited user. Please note that the password is automatically created.
+
+- **consul_nomad_autojoin_token generated by Nomad Server OCC** *(required)*: This token is generated by the [HashiCorp Nomad Cluster](/docs/products/tools/marketplace/guides/hashicorp-nomad-cluster/) deployment and can be found in the `~/.deployment_secrets.txt` file created in your original cluster.
```

Comment:
I might be missing it, but we should mention this somewhere earlier. Is here is the first place that mentions the other OCC?

Reply:
We did mention it earlier, but I called it out a little more obviously when editing the opening paragraph.